atomic_reactor/build.py

```diff
@@ -12,8 +12,10 @@
 import json
 
 import logging
+import docker.errors
+import atomic_reactor.util
```

Comment:
This seems to be only for `get_inspect_for_image`. Why not add that to the list of symbols we `from`...`import` from that module (on line 18)? Could also help with line wrapping in places where we call that function.

Reply:
that is just for unit tests, as I am unable to mock that function which is in different namespace for testing : test_parent_image_inspect, test_base_image_inspect